src/wrapper.js

```diff
@@ -5,10 +5,27 @@
  * Released under the MIT license
  * Date: @DATE
  */
-(function(document, window, $) {
-    'use strict';
+window = ( typeof global !== "undefined" ? global : typeof self !== "undefined" ? self : typeof window !== "undefined" ? window : {} );
+document = window.document || {};
```

Comment:
defining global variables `document` and `window` could have weird side-effects on other scripts. You should avoid this IMO